cfme/tests/infrastructure/test_vm_clone.py

```diff
@@ -34,7 +34,7 @@ def clone_vm_name():
 
 
 @pytest.fixture
-def create_vm(appliance, provider, request):
+def create_vm(appliance, provider, request, handle_cleanup_exception=True):
```

Comment:
I expect this will eventually be modified to use the global `create_vm`? Is this just a temporary fix? 

https://github.com/ManageIQ/integration_tests/blob/9874103d0967457b13f492c5be961613411ee3f6/cfme/fixtures/vm.py#L131-L138

Reply:
Whoops. I guess I was using the wrong create_vm??